LEGO1/orientableroi.h

```diff
@@ -0,0 +1,52 @@
+#ifndef ORIENTABLEROI_H
+#define ORIENTABLEROI_H
+
+#include "mxmatrix.h"
+#include "roi.h"
+
+class OrientableROI : public ROI {
+public:
+	// OFFSET: LEGO1 0x100a4420
+	OrientableROI()
+	{
+		FILLVEC3(m_world_bounding_box.Min(), 888888.8);
+		FILLVEC3(m_world_bounding_box.Max(), -888888.8);
+		ZEROVEC3(m_world_bounding_sphere.Center());
+		m_world_bounding_sphere.Radius() = 0.0;
+		ZEROVEC3(m_world_velocity);
+		IDENTMAT4(m_local2world.GetMatrix());
+	}
+	// OFFSET: LEGO1 0x100a4630 TEMPLATE
+	// OrientableROI::`scalar deleting destructor'
+
+	virtual const Vector3& GetWorldVelocity() const;
+	virtual const BoundingBox& GetWorldBoundingBox() const;
+	virtual const BoundingSphere& GetWorldBoundingSphere() const;
+	//	virtual float* GetWorldPosition() const { return (float*) m_local2world.GetData() + 12; }
+	//	virtual float* GetWorldDirection() const { return (float*) m_local2world.GetData() + 8; }
+	//	virtual float* GetWorldUp() const { return (float*) m_local2world.GetData() + 4; }
+
+protected:
+	// vtable + 0x14
+	virtual void VTable0x14() { VTable0x1c(); }
+	virtual void UpdateWorldBoundingVolumes() = 0;
+
+public:
+	virtual void OrientableROI::VTable0x1c();
+	// vtable + 0x20
+	virtual void OrientableROI::SetLocalTransform(const MxMatrix& p_transform);
```

Comment:
The class name should not be prefixed

Reply:
Oops, I'll fix this. Things always seem to get shuffled between the cpp and h while I work, and copy/paste errors like this creep in.